### DISCUSSION TITLE
Using media player groups

### DISCUSSION THREAD

**Discussion by moelloman** [keatontaylor/alexa-actions#165] [Q&A] [answered]
I followed the installation instructions and implemented the actionable notfications successfully. Many thanks for developing this amazing feature!
I created some automations and it works with a single media player  (I use Alexa devices). Then I tried to replace the single player by a media player group. Unfortunelly it doesn't work. I tried it with a Alexa device group and a media player group in HA - both don't give out any notification.
Any ideas or suggestions?

**Comment by DEADSEC-SECURITY** [comment, marked as answer]
Currently thats not supported. I would recommend you do a loop and trigger each one or manually trigger them individually

**Reply by moelloman** [reply]
Thx. May I add it to the issue list?

**Reply by DEADSEC-SECURITY** [reply]
Yhe go ahead
